package.json

```diff
@@ -66,10 +68,15 @@
     "mocha.js",
     "LICENSE"
   ],
+  "browser": {
+    "debug": "./lib/browser/debug.js",
```

Comment:
These (along with) replace the rewrites we were doing in [`support/compile.js`](https://github.com/mochajs/mocha/pull/1727/files#diff-4519ae56dd3583387155ed89f8a99ba1L45)


For more details see https://github.com/substack/node-browserify#browser-field